### DISCUSSION TITLE
No se ve el mapa de viaje ni la misión FrontEnd :(

### DISCUSSION THREAD

**Discussion by ValeriaGutierrezA** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#571] [Q&A]
Hola! 
A alguien también le está pasando que no se ve el mapa de viaje ni la misión FrontEnd como tal en el Teams? 
Si sí, saben por qué?:(

**Comment by doguedogue** [comment]
Exacto, no se ve 😌

tranquiz, mañana es la explicación de todo a las 7pm por Teams o por Instagram 🚀
